boards/xtensa/heltec_wifi_lora32_v2/heltec_wifi_lora32_v2.dts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Instituto de Pesquisas Eldorado (eldorado.org.br)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32.dtsi>
+
+/ {
+	model = "heltec_wifi_lora32";
+	compatible = "espressif,esp32";
+
+	aliases {
+		uart-0 = &uart0;
+		i2c-0 = &i2c0;
+		led0 = &led0;
+		sw0 = &button0;
+		vext = &vext;
```

Comment:
Do you really need this alias? Why not use `DT_NODELABEL(vext)` to get the node identifier?

Reply:
Works with DT_NODELABEL. Thanks!

Reply:
> > @mbolivar-nordic, I described the "vext control" and "oled reset" pins on the DTS under leds/gpio-leds. Do you have any suggestion of better place to describe it? Thanks!
> 
> Are they actually LEDs? Or are tehy something else?

They dont describe any LED. The "vext control" controls a mosftet that turns on/off  the power supply for devices external to the microcontroler  (like the onboard OLED display is included).
The "oled reset" pin is the RST pin of the OLED display.